feat(Authenticator): accept general JSX children (#1685)
aws-amplify/amplify-ui: .changeset/six-bikes-develop.md

---
"@aws-amplify/ui-react": patch
---

> Adjusts Authenticator to accept general JSX children. Now children no longer have to be a render prop.

Reviewer: Nit:

```suggestion
Adjusts Authenticator to accept both ReactNode as well as render prop children
```
